tests/run/unicode_imports.srctree

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# tag: py3, pep489
+
+PYTHON setup.py build_ext --inplace
+PYTHON -m mydoctest
+
+########### mydoctest.py #######
+
+import sys
+if (sys.version_info[0] < 3 or
+    (sys.version_info[0] == 3 and sys.version_info[1] < 5)):
+    exit() # I'm struggling to restrict srctree tests to a single Python version
```

Comment:
There is a blacklist named `VER_DEP_MODULES` in `runtests.py`.

Reply:
Good to know. In this case I don't actually want the blacklist since I do want to test the cythonize stage with lower Python versions (just not the C compilation or running). I've clarified the comment though